java/core/src/java/org/apache/orc/Reader.java

```diff
@@ -648,6 +653,24 @@ public Options allowPluginFilters(boolean allowPluginFilters) {
       this.allowPluginFilters = allowPluginFilters;
       return this;
     }
+
+    public int minSeekSize() {
```

Comment:
Could you add `@since` to newly added methods?

Reply:
Sorry missed that, added to the new methods in Options.